src/hooks/index.ts

```diff
@@ -1,5 +1,6 @@
 import { useAccountBalance } from './useAccountBalance';
 import { useAccounts } from './useAccounts';
+// eslint-disable-next-line import/no-cycle
```

Comment:
can we add a TODO or ticket to fix these?

Reply:
there are so many that I gave up. We need to get rid of all .index files and restructure quite a few imports to get rid of all the cycles in the codebase.